pkg/namespaced/client.go

```diff
@@ -0,0 +1,22 @@
+package namespaced
+
+import (
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func NewClient(client client.Client, namespace string) Client {
+	return Client{Client: client, Namespace: namespace}
+}
+
+type Client struct {
+	client.Client
+	Namespace string
```

Comment:
I see the namespace ATM is only used by the `signup_service` . I was wondering if it will be used in other places , otherwise we could stick to the `client.Client` type only and pass the namespace to the signupService only. Just an idea.

Reply:
Not only there,  it's used also via the `NamespacedName` method
https://github.com/codeready-toolchain/registration-service/blob/08d17055a7db2653f25fe72126a85be9a5b4a592/pkg/namespaced/client.go#L17-L22
in all cases when the code gets a resources from the namespace.

Reply:
Ok I see, I've missed that. then yes it makes sense to have a new struct holding also the namespace. thanks for the explanation.